rubi/tests/README.md

```diff
@@ -0,0 +1,68 @@
+# rubi-py testing suite
+
+This testing suite covers the rubi-py sdk's entire codebase. Our testing coverage is currently at 50%
+
+## Installation and Setup:
+
+To run the entire test suite, execute the following command in your terminal:
+
+```shell 
+    pytest rubi_tests.py
```

Comment:
i think it would be great if we could include these all within the `pyproject.toml` file utilizing [scripts](https://python-poetry.org/docs/pyproject#scripts)